### DISCUSSION TITLE
Publish to open vsx registry

### DISCUSSION THREAD

**Discussion by kishaningithub** [microsoft/vscode-cpptools#11077] [Ideas]
As a [VSCodium](https://vscodium.com/) user i would really love it if this extension was published to the [open vsx registry](https://open-vsx.org)

As you can see below this extension does not exist in open vsx as of now

Search query
https://open-vsx.org/?search=cpptools&category=&sortBy=relevance&sortOrder=desc

Screenshot
<img width="1412" alt="image" src="https://github.com/microsoft/vscode-cpptools/assets/763760/477a7af8-7f0d-4dff-b62a-238acca23ef9">

**Comment by bobbrow** [comment]
Thank you for your feedback.  Yes, we are aware of this.  Our license prohibits usage of the C++ extension outside of Microsoft's official version of VS Code so we cannot publish to OpenVSX.  We recommend using the official version of VS Code over VSCodium and other off-brand distros.

**Reply by kishaningithub** [reply]
From what i see https://github.com/microsoft/vscode-cpptools/blob/main/LICENSE.md this is just an MIT license. So you should not have any limitations IMO. Kindly correct if wrong.

**Reply by bobbrow** [reply]
The TypeScript code in this repository is MIT.  The binaries that are packaged into the VSIX have a different license and that was the one I was referring to.  Sorry about the confusion.

**Reply by bobbrow** [reply]
FWIW, this is explained in the actual license.md at the root of the repository. 

```
Additional binary files may be downloaded at runtime after the initial
installation; these are governed by the more restrictive proprietary license
terms found at [RuntimeLicenses](RuntimeLicenses).
```

The runtime licenses are found here: https://github.com/microsoft/vscode-cpptools/tree/main/RuntimeLicenses
